test/runner/lib/executor.py

```diff
@@ -276,7 +276,7 @@ def command_network_integration(args):
         remotes = [instance.wait_for_result() for instance in instances]
         inventory = network_inventory(remotes)
 
-        filename = 'test/integration/inventory.networking'
+        filename = 'test/integration/inventory.network'
```

Comment:
This inventory filename (and the following one) should not be changed.

The file `test/integration/inventory.network` already exists and is used by the old tests. The file `test/integration/inventory.networking` is used by `ansible-test` and is generated at the time tests are run.